Userland/ntpquery.cpp

```diff
@@ -100,27 +100,33 @@ static String format_ntp_timestamp(NtpTimestamp ntp_timestamp)
 
 int main(int argc, char** argv)
 {
-    if (pledge("stdio inet dns", nullptr) < 0) {
+    if (pledge("stdio inet dns settime", nullptr) < 0) {
         perror("pledge");
         return 1;
     }
 
-    // FIXME: Request serenityos.pool.ntp.org here https://manage.ntppool.org/manage/vendor
-    // and then use that as the default value.
-    // Until then, explicitly pass this as `ntpquery pool.ntp.org`.
-    const char* host = nullptr;
-
+    bool set_time = false;
+    // FIXME: Change to serenityos.pool.ntp.org once https://manage.ntppool.org/manage/vendor/zone?a=km5a8h&id=vz-14154g is approved.
+    const char* host = "time.google.com";
     Core::ArgsParser args_parser;
-    args_parser.add_positional_argument(host, "NTP server", "host", Core::ArgsParser::Required::Yes);
+    args_parser.add_option(set_time, "Adjust system time (requires root)", "set", 's');
+    args_parser.add_positional_argument(host, "NTP server", "host", Core::ArgsParser::Required::No);
     args_parser.parse(argc, argv);
 
+    if (!set_time) {
+        if (pledge("stdio inet dns settime", nullptr) < 0) {
+            perror("pledge");
+            return 1;
+        }
+    }
```

Comment:
Shouldn't this be:

```suggestion
    if (pledge("stdio inet dns settime", nullptr) < 0) {
        perror("pledge");
        return 1;
    }
```

Reply:
Oh, good catch. The `pledge("stdio inet dns settime", nullptr) < 0) ` is at the start of main(); this was supposed to drop settime if -s isn't passed. Now it does.

Reply:
Oh, I misunderstood what you were trying to do, but somehow still found a bug...